logos/tests/performance/analyze_eccs_ablation.py

```
@@ -0,0 +1,371 @@
+    print(f"{'='*60}")
+
+    total = stats.total_decisions
+    changed = stats.correction_changed_count
+    hit_rate = (changed / total * 100) if total > 0 else 0.0
+
+    print(f"\n  Decisions:          {total}")
+    print(f"  Correction changed: {changed} ({hit_rate:.1f}%)")
+    print(f"  Queued:             {stats.queued_count} ({stats.queued_count/max(1,total)*100:.1f}%)")
+
+    # Tier distribution
+    print(f"\n  Tier distribution (selected candidates):")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)